internal/datanode/l0_compactor.go

```diff
@@ -50,7 +52,7 @@ import (
 )
 
 type levelZeroCompactionTask struct {
```

Comment:
also move l0_compactor.go to datanode/compaction/ dir?

Reply:
YES, planning to do so.